src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -29,8 +31,15 @@ interface CommunityAPIService {
     return "$description for $contractTypeName Referral ${referral.referenceNumber} with Prime Provider $primeProviderName\n$url"
   }
 
+  fun getNotes(referral: SentReferralDTO, url: String, description: String, contractTypeName: String, primeProviderName: String): String {
+    return "$description for $contractTypeName Referral ${referral.referenceNumber} with Prime Provider $primeProviderName\n$url"
+  }
+
   fun setNotifyPPIfRequired(appointment: Appointment) =
     NO == appointment.attended!! || appointment.notifyPPOfAttendanceBehaviour ?: false
+
+  fun setNotifyPPIfRequired(deliverySession: DeliverySessionDTO) =
+    NO == deliverySession.sessionFeedback.attendance.attended!! || deliverySession.sessionFeedback.behaviour.notifyProbationPractitioner ?: false
```

Comment:
i'm not sure what the elvis operator is doing here. this should be a plain boolean expression.

```
NO == deliverySession.sessionFeedback.attendance.attended!! || deliverySession.sessionFeedback.behaviour.notifyProbationPractitioner == true
```